### DISCUSSION TITLE
legion 5 pro gen 8 amd,reduce fan curve in custom mode doesnt apply

### DISCUSSION THREAD

**Discussion by Valleron2014** [BartoszCichecki/LenovoLegionToolkit#1185] [General]
i have legion 5 pro gen 8 16ARX8.Im trying to reduce fan speed but legion toolkit doesnt apply the new curve that i create.Can only work when i increase fan speed.Its a bug?

**Comment by BartoszCichecki** [comment]
Duplicate of #1071
